pages/1.11/administering-clusters/replacing-a-master-node/index.md

```diff
@@ -4,43 +4,44 @@ navigationTitle:  Replace a master node
 title: Replace a master node
 menuWeight: 800
 excerpt: Replacing a master node in an existing DC/OS cluster
-
 enterprise: true
 ---
-
 <!-- This source repo for this topic is https://github.com/mesosphere/dcos-docs-site -->
+You can replace a master node in an existing DC/OS cluster. You should keep in mind, however, that you should only ever replace one master at a time. The following steps summarize how to replace a master node for a DC/OS cluster.
 
+# To replace a master node
+1. Back up ZooKeeper state information using Exhibitor, the Guano utility, or a custom script.
```

Comment:
We should just link to https://docs.mesosphere.com/1.11/installing/installation-faq/#q-how-do-i-backup-zookeeper-using-guano (mutatis mutandis for other versions)

Reply:
CC @timaa2k @adamtheturtle @jongiddy is that still the correct reference?

Reply:
I'm not sure but this will be changing very soon, right @timaa2k ?

Reply:
If the FAQ info for the back up steps are still valid, I can link to them. I'm not completely confident in whether they are accurate or legacy info. It raises some questions (for me anyway): is dcos-shell a standalone CLI package? how/when is it installed?

Reply:
It is the correct reference until we release a new way of backing up ZooKeeper.
At this point it is not clear whether we will backport the new way.